src/app/static/addresses.ts

```diff
@@ -62,7 +62,7 @@ export class Addresses {
   public static CRV_HUSD = '0x29780C39164Ebbd62e9DDDE50c151810070140f2'.toLowerCase();
```

Comment:
need to remove this class and use only data from the contract service

Reply:
👍 gradually getting through all the usages